models/metrics/metric_unique_users_count.sql

```diff
@@ -0,0 +1,62 @@
+{{ 
+    config(
+        materialized='table',
+        tags=['metrics'],
+        cluster_by=['metric_date']
+    )
+}}
+
+with
+
+min_address as (
```

Comment:
the logic here is only counting wallets only on the first day they are active?
not sure how useful this is. i think the block explorers, it's either looking at cumulative wallets over time, or things like daily, weekly, monthly active wallets, similar to that of monthly active user concept

Reply:
You can do that on the frontend.. but I guess we can add a cumulative column.